migrate.go

```diff
@@ -746,15 +747,19 @@ Check https://github.com/go-sql-driver/mysql#parsetime for more info.`)
 	table := dbMap.AddTableWithNameAndSchema(MigrationRecord{}, ms.SchemaName, ms.getTableName()).SetKeys(false, "Id")
 	//dbMap.TraceOn("", log.New(os.Stdout, "migrate: ", log.Lmicroseconds))
 
-	if dialect == "oci8" {
+	if dialect == "oci8" || dialect == "godror" {
 		table.ColMap("Id").SetMaxSize(4000)
 	}
 
 	err := dbMap.CreateTablesIfNotExists()
 	if err != nil {
 		// Oracle database does not support `if not exists`, so use `ORA-00955:` error code
 		// to check if the table exists.
-		if dialect == "oci8" && strings.HasPrefix(err.Error(), "ORA-00955:") {
+		if oerr := errors.Unwrap(err); oerr != nil {
+			err = oerr
+		}
+
+		if (dialect == "oci8" || dialect == "godror") && strings.HasPrefix(err.Error(), "ORA-00955:") {
```

Comment:
I don't think unwrapping is a good idea: there's no guarantee it unwraps to the point that you want it. Given that the previous code was a hack, perhaps this will just work:

```suggestion
		if (dialect == "oci8" || dialect == "godror") && strings.Contains(err.Error(), "ORA-00955:") {
```

Reply:
no. `strings.Contains(err.Error(), "ORA-00955:") ` is not work.   
 driver `godror`  and `oci8` have different errors.   oci8 return origin oracle client error[err.Eorrr() is "ORA-00955: xxxx "], godror return a wrap error[err.Eorrr() is "xxxx: ORA-00955: xxxx "].  so `strings.Contains(err.Error(), "ORA-00955:")` is not work in godror.

Reply:
Why wouldn't a `Contains()` work there?

Reply:
ohhhhh, `Contains()` is good. I see it as `HasPrefix `